network/p2p/acp118/handler_test.go

```diff
@@ -26,25 +26,23 @@ func TestHandler(t *testing.T) {
 	tests := []struct {
 		name         string
 		cacher       cache.Cacher[ids.ID, []byte]
-		verifier     Verifier
+		verifierErrs []*common.AppError
 		expectedErrs []error
 	}{
 		{
 			name:   "signature fails verification",
 			cacher: &cache.Empty[ids.ID, []byte]{},
-			verifier: &testVerifier{
-				Errs: []*common.AppError{
-					{Code: 123},
-				},
+			verifierErrs: []*common.AppError{
+				{Code: 123},
 			},
 			expectedErrs: []error{
 				&common.AppError{Code: 123},
 			},
 		},
 		{
-			name:     "signature signed",
-			cacher:   &cache.Empty[ids.ID, []byte]{},
-			verifier: &testVerifier{},
+			name:         "signature signed",
+			cacher:       &cache.Empty[ids.ID, []byte]{},
+			verifierErrs: nil,
```

Comment:
This line can be deleted entirely iirc because the zero value of a slice is safe to use